config/locales/en/nsm/work_items.yml

```diff
@@ -125,7 +125,7 @@ en:
         nsm/work_item_form:
           attributes:
             base:
-              no_change: The new values cannot be the same as the values submitted by the provider
+              no_change: The new values cannot be the same the previously submitted values.
```

Comment:
We're going to need to do something similar for the four no_change strings in letters_and_calls.yml (there are four because there are 2 forms, and for each form we have the error message in 2 places depending on whether there is an uplift field or not)

Reply:
Ah I was searching for the wrong words, amending